lib/rack/lint.rb

```diff
@@ -308,7 +312,8 @@ def check_environment(env)
         end
 
         ## The <tt>SERVER_NAME</tt> must be a valid authority as defined by RFC7540.
-        unless (URI.parse("http://#{env[SERVER_NAME]}/") rescue false)
+        server_name = assert_required(SERVER_NAME)
+        unless (URI.parse("http://#{server_name}/") rescue false)
           raise LintError, "#{env[SERVER_NAME]} must be a valid authority"
```

Comment:
Up to you, but you could use `server_name` here (with similar changes it other error messages.

Reply:
Good idea, done.